modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/config/SourceWithoutConnectedOperationTestCase.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.config;
+
+import org.mule.functional.junit4.ExtensionFunctionalTestCase;
+import org.mule.tck.probe.JUnitLambdaProbe;
+import org.mule.tck.probe.PollingProber;
+import org.mule.test.vegan.extension.HarvestPeachesSource;
+import org.mule.test.vegan.extension.VeganExtension;
+
+import org.junit.Test;
+
+public class SourceWithoutConnectedOperationTestCase extends ExtensionFunctionalTestCase
+{
+    @Override
+    protected String getConfigFile()
+    {
+        return "vegan-config.xml";
+    }
+
```

Comment:
you need to override the `doSetUpBeforeMuleContextCreation` method and set `HarvestPeachesSource.isConnected`  to `false`